init_validators.go

```diff
@@ -209,6 +209,38 @@ func initVal(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
 
 	runenv.RecordMessage("starting........")
 
+	nodeId, err := cmd.GetNodeId(home)
+	if err != nil {
+		return err
+	}
+	_, err = client.Publish(
+		ctx,
+		testkit.ValidatorPeerTopic,
+		&appkit.ValidatorNode{
+			PubKey: nodeId,
+			IP:     config.IPv4.IP},
+	)
+	if err != nil {
+		return err
+	}
+
+	valCh := make(chan *appkit.ValidatorNode)
+	client.Subscribe(ctx, testkit.ValidatorPeerTopic, valCh)
+
+	var persPeers []string
+	for i := 0; i < runenv.TestGroupInstanceCount; i++ {
+		val := <-valCh
+		runenv.RecordMessage("Validator Received: %s, %s", val.IP, val.PubKey)
+		if !val.IP.Equal(config.IPv4.IP) {
+			persPeers = append(persPeers, fmt.Sprintf("%s@%s", val.PubKey, val.IP.To4().String()))
+		}
+	}
+
+	err = appkit.AddPersistentPeers(configPath, persPeers)
```

Comment:
This should be removed after we get this done https://github.com/celestiaorg/test-infra/issues/40